designer/server/src/common/components/sorting/template.njk

```diff
@@ -0,0 +1,49 @@
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% if params.sorting %}
+<form method="get" action="/library" aria-label="Sort forms">
+  <div class="app-field-group">
+    <div class="app-field-group__item">
+      {{ govukSelect({
+        id: "sort-select",
+        name: "sort",
+        label: {
+          text: "Sort by",
+          classes: "app-field-group__label"
+        },
+        classes: "app-field-group__input",
+        items: [
+          {
+            text: "Updated (newest)",
+            value: "updatedDesc",
+            selected: params.sorting.sortBy == "updatedAt" and params.sorting.order == "desc"
+          },
+          {
+            text: "Updated (oldest)",
+            value: "updatedAsc",
+            selected: params.sorting.sortBy == "updatedAt" and params.sorting.order == "asc"
+          },
+          {
+            text: "Form name (A to Z)",
+            value: "titleAsc",
+            selected: params.sorting.sortBy == "title" and params.sorting.order == "asc"
+          },
+          {
+            text: "Form name (Z to A)",
+            value: "titleDesc",
```

Comment:
I ended up changing this to camelCase so it is in-line with the pagination query params.